stablehlo/tests/interpret_abs.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: abs_op_test_si64
+func.func @abs_op_test_si64() -> tensor<3xi64> {
+  %operand = stablehlo.constant dense<[-2, 0, 2]> : tensor<3xi64>
+  %result = stablehlo.abs %operand : tensor<3xi64>
+  func.return %result : tensor<3xi64>
+  // CHECK-NEXT: tensor<3xi64>
+  // CHECK-NEXT: 2 : i64
+  // CHECK-NEXT: 0 : i64
+  // CHECK-NEXT: 2 : i64
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: abs_op_test_f64
+func.func @abs_op_test_f64() -> tensor<3xf64> {
+  %operand = stablehlo.constant dense<[23.1, -23.1, -1.1]> : tensor<3xf64>
```

Comment:
may be one positive , one negative an done zero just as the integer test looks like. Even negative zero is fine.